app/code/Magento/ProductOverrideDataExporter/etc/query.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_QueryXml:etc/query.xsd">
+    <query name="productPriceOverrider">
+        <source name="catalog_product_index_price">
+            <attribute name="entity_id" alias="productId" />
+            <attribute name="customer_group_id" alias="customerGroupCode" sort="asc"/>
+            <attribute name="website_id" sort="asc"/>
+            <attribute name="tax_class_id"/>
+            <attribute name="price" alias="regularPrice"/>
+            <attribute name="final_price" alias="finalPrice"/>
+            <attribute name="min_price"/>
+            <filter glue="and">
+                <condition attribute="entity_id" operator="in" type="placeholder">entityIds</condition>
```

Comment:
I'm a little confused on how these get populated. How is the placeholder actually used in calling this?